examples/typescript/local_node.ts

```diff
@@ -18,6 +18,7 @@ async function runLocalNode() {
     console.log("starting local node...");
     localNode = new cli.LocalNode({ showStdout: false });
     await localNode.run();
+    console.log("local node started");
```

Comment:
Intentionally included?

Reply:
yea.. just to give more info on what is going on, especially because we dont show stdout